exercise/staff_views.py

```diff
@@ -168,7 +168,7 @@ class InspectSubmissionView(SubmissionBaseView, BaseFormView):
     def get_common_objects(self) -> None:
         super().get_common_objects()
         self.get_summary_submissions()
-        self.has_files = self.submission.files.count() > 0
+        self.has_files = self.submission.files.exists()
```

Comment:
If I remember correctly, the html template queries the database again to retrieve the submitted file objects. There would be less queries overall if the files are queried and saved to a variable here. Though, I suppose the template does not query anything if `has_files` is `False`.

Reply:
Good catch. I pushed a new commit that optimizes this.